package.json

```diff
@@ -42,9 +42,7 @@
     "dev:ui": "(cd apps/ui && npm run dev)"
   },
   "deplint": {
-    "files": [
-      "apps/server/lib/**/*.ts"
-    ],
+    "files": [],
```

Comment:
We shouldn't be skipping lint here, I'd be interested in what you are trying to avoid by removing this.

Reply:
@joshbwlng see https://jel.ly.fish/thread-70e3970b-c722-4341-989f-a74278835e92

Reply:
@karaxuna Right, I think it would be better to change the way the code is organized instead of disabling lint. One idea is to move `plugin-jellyfish` to be an actual plugin in its own repo like `jellyfish-plugin-oauth`. You may end up with the `hydraAdmin()` code in two places, but I think it would be a simpler approach that better matches how we manage other plugins.

Reply:
I can help you set up a `jellyfish-plugin-oauth` repo, might go ahead and start that today if I have the time.

Reply:
Created the repo and started bringing in your work here: https://github.com/product-os/jellyfish-plugin-oauth/pull/1
Need to fix the two todos I've commented on in the integration code, but I think everything else should be good once we solve that.